airrohr-firmware/PCF8574.cpp

```diff
@@ -241,6 +241,10 @@ void PCF8574::begin()
     lastReadMillis = millis();
 }
 
+void PCF8574::setAddress(uint8_t address){
```

Comment:
@Warukira did you add this as a patch, we should not be adding any extra functions to the core lib. This is already covered in the constructur.

Reply:
Yes, I did @chegejames. Working on it.